test/index.spec.js

```diff
@@ -25,7 +25,7 @@ describe('babel-plugin-css-modules-transform', () => {
     }
 
     function readExpected(path) {
-        return readFileSync(resolve(__dirname, path), 'utf8');
+        return readFileSync(resolve(__dirname, path), 'utf8').trim();
```

Comment:
@leoasis what do you mean? Adding the regex for query params mentioned by @jvivs (https://github.com/michalkvasnicak/babel-plugin-css-modules-transform/pull/11#discussion_r64774154)? Because trimming is already merged so I don't understand this comment. :)